tests/e2e/vLLM/recipes/actorder/recipe_w4a16_actorder_weight_dampfrac1e-1.yaml

```diff
@@ -0,0 +1,15 @@
+quant_stage:
+  quant_modifiers:
+    GPTQModifier:
+      dampening_frac: 0.1
```

Comment:
Just make sure that, for qwen, the inputs and model quantization looks correct. Hessian inversion issues can sometimes be the result of bad data or trying to quantize modules you didn't mean to quantize

Reply:
This may no longer be necessary for the larger model. I'll try it quick

Reply:
This seems to work fine now with the larger model. I got rid of this recipe and switched back to `recipe_w4a16_actorder_weight.yaml`

Reply:
Okay, as long as we know that this recipe is working as expected

Reply:
It's the same recipe, except for dampening_frac

Reply:
Removed this in favor of original recipe that doesn't set dampening_frac from default. Things seem to be running fine for the larger model